common/utils.py

```diff
@@ -33,7 +33,8 @@ def clean(self, value_list):
 
     def clean_row(self, row):
         for item in IGNORED_LIST:
-            del row[item]
+            if item in row:
```

Comment:
use select column name instead of select * ,check this in payment,wallet and orchestrator

Reply:
In wallets we are not using Select *